src/stylesheets/elements/_inputs.scss

```diff
@@ -1,12 +1,20 @@
 .usa-select {
   @include add-background-svg('sam/arrow-down');
   background-size: 0.6rem;
+
+  &.usa-select--small {
+    @include u-font('sans', '3xs');
+    @include u-margin-top(0.5);
+    height: 2.1rem;
+  }
 }
 
```

Comment:
could you move tag related css to `src/stylesheets/elements/_tags.scss`

Reply:
I moved the display related classes out of inputs